### PR TITLE
release: v.1.0.0-beta 1

### PR DESCRIPTION
**Pull request by 32penkin**

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -0,0 +1,85 @@
+## <small>0.0.1 (2019-05-30)</small>
```
Comment (artyorsh) [inline CHANGELOG.md]: No need to generate changelog for the first release

File: package.json [hunk 2]
```diff
@@ -20,10 +20,14 @@
     "ci:lint": "npm run lint",
     "ci:build": "npm run build",
     "ci:test": "npm run test",
-    "release": "npm run lint && npm run test",
+    "clean": "rimraf ./dist",
     "version:bump": "gulp version",
+    "conventional-changelog": "conventional-changelog",
     "version:changelog": "npm run conventional-changelog -- -i CHANGELOG.md -s",
-    "conventional-changelog": "conventional-changelog"
+    "publish": "./scripts/publish.sh",
```
Comment (artyorsh) [inline package.json]: I think it's better to perform checks here and remove `publish:release` script to avoid possible troubles if running `npm publish`

**Comment by artyorsh** [comment]
My suggestion is to rebase commit history in this PR, so we can merge 4 clean commits.

1-5 commits could be squashed into one (let's say "refactor(package): package structure")
8-12 commits could be squashed into one ("package(script): add package release scripts")

**Comment by artyorsh** [comment]
#2 and #3 can be closed

File: CHANGELOG.md [hunk 3]
```diff
@@ -0,0 +1,260 @@
+## 1.0.0-beta.1 (2019-05-31)
+
+* feat(build): build pipeline add ([482c466](https://github.com/eva-design/eva/commit/482c466))
```
Comment (nnixaa) [inline CHANGELOG.md]: there are two versions in the changelog with repeating commits, remove 0.0.1 version and leave 1.0.0-beta.1 only

File: DEV_DOCS.md [hunk 4]
```diff
@@ -0,0 +1,17 @@
+# Release
+
+To start a new release (publish the framework packages on NPM) you need:
+
+1. Create a new release branch called `release:v1.0.2`
+2. `npm run release:validate` - this will check linter issues and tests.
+3. `npm run release:prepare` - this will create ready for publishing packages in `./dist`.
+4. MANUALLY update a version in main ./package.json to a new one.
+5. `npm run version:bump` - this will update versions of `dss`, `eva`, `processor` packages.
+6. Update version in `package-lock.json` and `packages/processor/package-lock.json`.
```
Comment (nnixaa) [inline DEV_DOCS.md]: there shouldn't be any additional package-lock files except for the main one.

File: CHANGELOG.md (hunk 1 above)
Comment (32penkin) [reply CHANGELOG.md]: Done.

**Pull request merged**